Packs/Core/ReleaseNotes/1_3_21.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Investigation & Response
+- Fixed an issue in ***xdr-blacklist-files*** and ***xdr-whitelist-files***.
```

Comment:
Please give more details. maybe `Fixed an issue where the ***xdr-blacklist-files*** and ***xdr-whitelist-files*** deprecated commands were mistakenly removed.`